### PR TITLE
Add cflinuxfs3 to scf

### PR DESCRIPTION
**Pull request by lightlace**
supersedes #2013

### CODE REVIEW

File: container-host-files/etc/scf/config/role-manifest.yml [hunk 1]
```diff
@@ -1285,7 +1289,6 @@ instance_groups:
           memory: 2800
           virtual-cpus: 4
           healthcheck:
-            readiness:
```
Comment (jimmykarily) [inline container-host-files/etc/scf/config/role-manifest.yml]: rebase on develop so you don't need this

Comment (jimmykarily) [reply container-host-files/etc/scf/config/role-manifest.yml]: Search this line `    properties.cflinuxfs2-rootfs.trusted_certs: '"((ROOTFS_TRUSTED_CERTS))((#ROOTFS_TRUSTED_CERTS))\n((/ROOTFS_TRUSTED_CERTS))((INTERNAL_CA_CERT))"'
` in this file (~2178). There is equivalent for cflinuxfs3 but there is one for sle and opensuse stacks. I think you need one for cflinuxfs3 as well.

File: src/scf-release/src/acceptance-tests-brain/test-scripts/012_minibroker_redis_test.sh [hunk 2]
```diff
@@ -104,6 +104,7 @@ function wait_for_namespace() {
             select(.reason != "PodCompleted")
         '
         local ret=$?
+
```
Comment (jimmykarily) [inline src/scf-release/src/acceptance-tests-brain/test-scripts/012_minibroker_redis_test.sh]: please avoid irrelevant changes

**Comment by lightlace** [comment]
superseded by #2038.

**Pull request closed without merging**